lib/mutant/ast/regexp/transformer.rb

```diff
@@ -0,0 +1,82 @@
+module Mutant
+  module AST
+    module Regexp
+      class Transformer
+        include AbstractType
+
+        REGISTRY = {} # rubocop:disable MutableConstant
```

Comment:
> I think the Registry should just be reduced down and the call should be moved to Mutator.call which assumes a REGISTRY constant

I had this in a local branch for cleaning up the mutators but never pushed it so far.